make-all.sh

```diff
@@ -27,3 +27,12 @@ fi
 cp src/TerminalFighter/TerminalFighter ../
 ./test/runTests
 cd ..
+
+# Update docker if there are any new files
+docker build . -t terminal_documentation
```

Comment:
The image name should be a variable. And we should prefix the org name.

`IMAGE_NAME=JeLLyNinjas/TerminalFighterDocs` is my suggestion